core/api/core.api

```diff
@@ -6401,8 +6401,7 @@ public final class dev/kord/core/entity/GuildWidget : dev/kord/core/KordObject,
 }
 
 public abstract class dev/kord/core/entity/Icon : dev/kord/core/KordObject {
-	public synthetic fun <init> (ZLdev/kord/rest/route/CdnUrl;Ldev/kord/core/Kord;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
-	public final fun getAnimated ()Z
```

Comment:
Could you bring this back for binary compatibility? Not sure if it should be deprecated though.